### PR TITLE
Added roman-numeral-converter by h2o-Noob

### PR DESCRIPTION
**Pull request by h2o-Noob**
## Description

- My web-app converts any number from 1 to 999,999,999 to its roman-numeral form. I didn't used any maths library or api request. Whole algo was written using pure vanilla-js.

## Add GIF/ScreenShot 

- https://drive.google.com/file/d/11z3qHQKAJXS0QJz5AZmVYc_MweZ64_gG/view?usp=sharing

## Task Checklist
Put x in between the square brackets without any spaces to tick your Task.  Create a PR only after you complete all the tasks.
- [x] You have created a new branch for contribution.
- [x] Added README.md for your project in a specified folder.
- [x] Add Demo in GIF or Screenshot form.
- [x] You are Registered Officially for HacktoberFest 2021 as Partcipant.
- [x] Followed Contribution Guidelines.

## Issue 
Mention your issue no.
64

### CODE REVIEW

**Comment by h2o-Noob** [comment]
@Harshil-Jani sir I created a separate branch now, please merge my request :)

File: romanNumeralConverter_h2o-Noob/index.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+            </table>
+        </div>
+    </div>
+    <footer>
+        &copy noobCreates.org
+    </footer>
+    
+    <script src="script.js"></script>
+    </body>
+</html>
```
Comment (soumik2012) [inline romanNumeralConverter_h2o-Noob/index.html:117]: Add new line at EOF

File: romanNumeralConverter_h2o-Noob/readme.txt [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Roman Numeral Converter Application
+
```
Comment (soumik2012) [inline romanNumeralConverter_h2o-Noob/readme.txt]: Add picture of project in gif

File: romanNumeralConverter_h2o-Noob/romCon.css [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+}
+
+form input[type=button]:active
+{
+    background-color: rgb(255, 94, 20);
+    border-radius: 10px;
+    border-top: 2px solid rgb(179, 57, 1);
+    border-left: 2px solid rgb(179, 57, 1);
+    box-shadow: 2px 2px 10px #08080869;
+}
```
Comment (soumik2012) [inline romanNumeralConverter_h2o-Noob/romCon.css:108]: Add new line at EOF

File: romanNumeralConverter_h2o-Noob/style.css [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+
+.rft
+{
+        border: 3px solid rgb(217 110 1);
+        border-spacing: 2px;
+        font-size: 40px;
+        text-align: center;
+        background-color: rgb(255, 107, 38);
+}
```
Comment (soumik2012) [inline romanNumeralConverter_h2o-Noob/style.css:131]: Add new at line at EOF

**Review by soumik2012 — CHANGES_REQUESTED** [review]
Everything is nice but some minnor changes are needed

**Comment by h2o-Noob** [comment]
> Everything is nice but some minnor changes are needed  

Sir now please merge my pr

**Pull request merged**